src/src/AdminBundle/Form/AdminCreateForm.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AdminBundle\Form;
+
+use AdminBundle\Entity\Admin;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\RepeatedType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+
+class AdminCreateForm extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('email', EmailType::class)
+            ->add('password', RepeatedType::class, array(
+                'type' => PasswordType::class,
+                'invalid_message' => 'The password fields must match.',
+                'options' => array('attr' => array('class' => 'password-field')),
+                'required' => true,
+                'first_options' => array('label' => 'Password'),
+                'second_options' => array('label' => 'Repeat Password'),
```

Comment:
Preco sme niekde pouzili repository a tu sme pouzili form buildre? Neda sa to pouzit nejako jednotne? Alebo su to 2 rozne veci?

Reply:
Oki